public/css/style.css

```
@@ -188,4 +188,97 @@ td.description {
 
 tbody.table_row {
     text-align: left;
+}
+
+div.register_container {
```

In css hanteren wij altijd - en geen _ 
```suggestion
div.register-container {
```